neuror/error_annotation.py

```diff
@@ -0,0 +1,79 @@
+"""Annotation errors on morphologies."""
```

Comment:
The content of this file could go in `sanitize.py`.

Since sanitize.py already expose `sanitize` and `sanitize_all`, I'd propose the names:
`annotate_neurolucida` and `annotate_neurolucida_all`